projects/architecture/interfaces/primitives/blocks.proto

```diff
@@ -1,15 +1,21 @@
 syntax = "proto3";
 
-import "./transactions.proto";
+import "transactions.proto";
 
 message BlockHeader {
-  uint32 version = 1;
-  uint32 id = 2;
-  uint32 prevBlockId = 3;
+    uint32 version = 1;
```

Comment:
You've accidentally shifted the indent from 2 to 4 (it's usually 2, see here: https://developers.google.com/protocol-buffers/docs/style)